.github/workflows/examples.yaml

```
@@ -32,3 +33,15 @@ jobs:
 
       - name: Run build
         run: pnpm turbo --filter ./examples/${{ matrix.dir }} build
+
+      - name: Install playwright
+        run: |
+          cat ./examples/${{ matrix.dir }}/package.json | 
+          if grep --silent '"@playwright/test"';
+            then pnpm dlx playwright install --with-deps;
+          fi
+
+      - name: Run E2E Tests (if exists)
+        run: pnpm turbo --filter ./examples/${{ matrix.dir }} test
+        env:
+          UPLOADTHING_SECRET: ${{ secrets.UPLOADTHING_TEST_SECRET }}
```

might want a different secret here linked to another app. 

there could be race conditions if these tests runs at the same time

---

Agreed. We should also put these apps in an org so we can all access them easily. 

```suggestion
          UPLOADTHING_SECRET: ${{ secrets.UPLOADTHING_E2E_TEST_SECRET }}
```